rust_src/src/buffers.rs

```diff
@@ -333,6 +378,105 @@ impl LispBufferRef {
         }
         self.local_flags[idx] = val;
     }
+
```

Comment:
Perhaps this should be a method on `LispBuffeRef` that allocates followed by something like this `create_new` which makes it ready for use?